2255.count-prefixes-of-a-given-string.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/count-prefixes-of-a-given-string/
+ */
+
+/**
+ * @param {string[]} words
+ * @param {string} s
+ * @return {number}
+ */
+var countPrefixes = function(words, s) {
+    
+    const changeToArr = s.toString("");
+    
+    //Answer
+    let matchCount = 0;
+    
+    //Deconstruct string in s array
+    const newArrBox = [];
+    let i = 0;
+    while (i < changeToArr.length) {
+        newArrBox.push(changeToArr.slice(0,(i + 1)));
+        i++;
+    }
```

Comment:
```suggestion
    for (let i = 1; i <= s.length; i++) {
        newArrBox.push(s.slice(0, i));
    }
```
You can also use the for loop.